components/menu/menu-item-styles.js

```diff
@@ -44,4 +44,23 @@ export const menuItemStyles = css`
 	:host([last]:hover) {
 		border-bottom-color: var(--d2l-color-celestine);
 	}
+
+	.d2l-menu-item-text {
+		flex: auto;
+		line-height: 1rem;
+		overflow-x: hidden;
+		overflow-y: hidden;
+		text-overflow: ellipsis;
+		white-space: nowrap;
+	}
```

Comment:
This replaces the generic `span` CSS duplicated in `menu-item.js` and `menu-item-selectable-styles.js`.